pkg/features/kube_features.go

```diff
@@ -700,7 +700,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSIMigrationAzureFileComplete:  {Default: false, PreRelease: featuregate.Alpha},
 	CSIMigrationvSphere:            {Default: false, PreRelease: featuregate.Beta},
 	CSIMigrationvSphereComplete:    {Default: false, PreRelease: featuregate.Beta},
-	RunAsGroup:                     {Default: true, PreRelease: featuregate.Beta},
+	RunAsGroup:                     {Default: true, PreRelease: featuregate.GA},
```

Comment:
when moving features to GA, we should:
1. set `LockToDefault: true` and add a comment tracking removal in 2 releases
2. remove checks for `.Enabled(features.RunAsGroup)` (since it is locked to true)
3. ensure API docs no longer indicate it is a beta feature (I think this is already done for this field)
4. ensure e2e test coverage exists